airbyte-db/db-lib/src/main/java/io/airbyte/db/PostgresUtils.java

```diff
@@ -17,9 +17,9 @@
 public class PostgresUtils {
 
   public static PgLsn getLsn(final JdbcDatabase database) throws SQLException {
-    // pg version 10+.
+    // pg version >= 10. For versions < 10 use query select * from pg_current_xlog_location()
```

Comment:
I see. Do we have any plans of supporting version < PG 10 for CDC workflows?

Reply:
I have not seen any issues till yet + Support for Postgres versions < 10 has already ended (see [this](https://endoflife.date/postgresql)) so I doubt anyone would be using it right now. I just put the comment that in case someone raises an issue with 9.4 we know what needs to be done.